docs/deployment_guide/partner_editable/troubleshooting.adoc

```diff
@@ -1,6 +1,10 @@
 // Add any unique troubleshooting steps here.
 
-For troubleshooting common Quick Start issues, refer to the https://fwd.aws/rA69w?[AWS Quick Start General Information Guide^] and https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html[Troubleshooting CloudFormation^].
+* For troubleshooting common Quick Start issues, refer to the https://fwd.aws/rA69w?[AWS Quick Start General Information Guide^] and https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html[Troubleshooting CloudFormation^].
+* If your AWS Organizations is new, you might run into the below issue for and you would need to retry the deployment in 24 hours.
```

Comment:
If your AWS Organizations is new, you might run into the below issue for Security Hub Centralized Configuration and you would need to retry the deployment in 24 hours.